app/models/user.js

```diff
@@ -0,0 +1,39 @@
+'use strict';
+
+module.exports = (sequelize, DataTypes) => {
+  const User = sequelize.define(
+    'user',
+    {
+      id: {
+        primaryKey: true,
+        type: DataTypes.INTEGER,
+        allowNull: false,
+        unique: true,
+        autoIncrement: true
+      },
+      first_name: {
```

Comment:
Set fields in models as camelCase

Reply:
I had interpreted it as snake_case. Thanks for the feedback